content/en/blog/_posts/2022-4-26-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,99 @@
+
+
+## Major Themes
+
+### Dockershim Removed from kubelet
+After an initial deprecation in v1.20, the dockershim has been removed from the kubelet. From v1.24 and up, if you are currently relying on Docker Engine as your container runtime, you will need to either use one of the other supported runtimes (such as containerd or CRI-O) or use cri-dockerd. For more information about ensuring your cluster is ready for this removal, please see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/). 
+
+### gRPC Probes Graduates to Beta
+With Kubernetes 1.24, the gRPC probes functionality has entered beta and is available by default.
+
+You can now configure startup, liveness, and readiness probes for your gRPC app without exposing an HTTP endpoint or using an extra executable, natively within Kubernetes.
+
+### Kubelet Credential Provider graduates to Beta
+Released as Alpha in Kubernetes v1.20, the Kubelet Credential Provider has now graduated to Beta. This allows the kubelet to dynamically retrieve credentials for a container image registry using exec plugins, communicating through stdio using Kubernetes versioned APIs, rather than storing them statically on disk.
+
+### Contextual Logging in Alpha
+Kubernetes v1.24 has introduced Contextual logging that enables the caller of a function to control all aspects of logging (output formatting, verbosity, additional values and names).
+
+### Avoiding collisions in IP allocation to Services
+A Service `ClusterIP` can be assigned:
+
+* dynamically, the cluster will automatically pick a free one within the configured Service IP range.
+* statically, the user will set one IP within the configured Service IP range.
+
+Service `ClusterIP` are unique, hence, trying to create a Service with a `ClusterIP` that has already been allocated will return an error.
+
+Kubernetes 1.24 introduces a new Feature Gate `ServiceIPStaticSubrange` that allows to use a different IP allocation strategy for Services, reducing the risk of collision.
+
+
+## Other Updates
+### Graduated to Stable
+* To do: Add all enhancements that graduated to stable
+
+### Major Changes
+* To do: Add information about Dockershim
+
+### Release Notes
+Check out the full details of the Kubernetes 1.24 release in our [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+    
+### Availability
+
+Kubernetes 1.24 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters using Docker container “nodes” with [kind](https://kind.sigs.k8s.io/). You can also easily install 1.24 using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
```

Comment:
`kind` supports more than one container tool, and also I think it might help not to mention Docker here:

```suggestion
Kubernetes 1.24 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.0).
To get started with Kubernetes, check out these [interactive tutorials](/docs/tutorials/) or run local 
Kubernetes clusters using containers as “nodes”, with [kind](https://kind.sigs.k8s.io/).
You can also easily install 1.24 using [kubeadm](/docs/setup/independent/create-cluster-kubeadm/).
```

Any reason we don't also mention minikube or just link to https://kubernetes.io/docs/tasks/tools/ and let readers pick?